mscore/pagesettings.cpp

```diff
@@ -108,159 +105,213 @@ void PageSettings::setScore(Score* s)
 
       clonedScore->doLayout();
       preview->setScore(clonedScore);
-      buttonApplyToAllParts->setEnabled(!cs->isMaster());
-      updateValues();
-      updatePreview(0);
+      applyToParts->setEnabled(!cs->isMaster());
+      updateWidgets();
+      updatePreview();
       }
 
 //---------------------------------------------------------
-//   blockSignals
+//   blockSignals - helper for updateWidgets
 //---------------------------------------------------------
 
 void PageSettings::blockSignals(bool block)
       {
-      for (auto w : { oddPageTopMargin, oddPageBottomMargin, oddPageLeftMargin, oddPageRightMargin,
-         evenPageTopMargin, evenPageBottomMargin, evenPageLeftMargin, evenPageRightMargin, spatiumEntry,
-         pageWidth, pageHeight } )
+      for (auto w : { oddTopMargin, oddBottomMargin, oddLeftMargin, oddRightMargin,
+                     evenTopMargin,evenBottomMargin,evenLeftMargin,evenRightMargin,
+                     staffSpace } )
             {
             w->blockSignals(block);
             }
-      pageOffsetEntry->blockSignals(block);
-      pageGroup->blockSignals(block);
+      twosided->blockSignals(block);
+      typesList->blockSignals(block);
+      sizesList->blockSignals(block);
+      unitsList->blockSignals(block);
+      portrait->blockSignals(block);
+      landscape->blockSignals(block);
+      pageOffset->blockSignals(block);
       }
 
 //---------------------------------------------------------
-//   setMarginsMax
+//   updatePreview
 //---------------------------------------------------------
 
-void PageSettings::setMarginsMax(double pw)
+void PageSettings::updatePreview()
       {
-      oddPageLeftMargin->setMaximum(pw);
-      oddPageRightMargin->setMaximum(pw);
-      evenPageLeftMargin->setMaximum(pw);
-      evenPageRightMargin->setMaximum(pw);
+      preview->score()->doLayout();
+      preview->layoutChanged();
       }
 
 //---------------------------------------------------------
-//   updateValues
-//    set gui values from style
+//   updateWidgets
```

Comment:
there is no need to rename the method. Why? It doesn't update widgets, it updates values in the controls.

Reply:
@sidewayss please address the comment.

Reply:
And it updates much more than the values.  It updates all the configured settings for each widget.